utils/check_task_guides.py

```diff
@@ -0,0 +1,119 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import importlib.util
+import os
+
+
+# All paths are set with the intent you should run this script from the root of the repo with the command
+# python utils/check_task_guides.py
+TRANSFORMERS_PATH = "src/transformers"
+PATH_TO_TASK_GUIDES = "docs/source/en/tasks"
+REPO_PATH = "../../../Downloads"
```

Comment:
This doesn't sound right :sweat_smile: Also, is it used?

Reply:
You're right, it's not needed here:) I'll remove it